index/cache.py

```diff
@@ -0,0 +1,9 @@
+import boto3
+import scheduler as schedule
+import os
+
+
+def exports():
+    dynamodb = boto3.client('dynamodb')
+    scheduler = schedule.Scheduler()
+    return dynamodb, scheduler
```

Comment:
In order to truly cache and reuse objects, you should first check if they exist globally or locally:
```
if var not in locals() and var not in globals():
   #import/instantiate object here
```
Otherwise you're still importing and creating each object every time the code runs.

Note that Python variables are [scoped to the nearest function, class, or module](https://stackoverflow.com/a/2829642), you can use a bunch of if statements in the cache and that will work just fine.